app/utils/translations.js

```diff
@@ -0,0 +1,1011 @@
+            },
+            "4": {
+                "items": {
+                    "11": {
+                        "label": "I was interested in what someone had to say"
+                    },
+                    "10": {
+                        "label": "I was physically animated; moved around."
+                    },
+                    "13": {
+                        "label": "I acted playful."
+                    },
+                    "12": {
+                        "label": "I sought advice."
+                    },
+                    "15": {
+                        "label": "I spoke in a loud voice."
+                    },
+                    "14": {
+                        "label": "I expressed self-pity or feelings of victimization."
+                    },
+                    "16": {
+                        "label": "I exhibited a high degree of intelligence."
+                    },
+                    "1": {
+                        "label": "I tried to control the situation."
+                    },
+                    "3": {
+                        "label": "I behaved in a competitive manner."
+                    },
+                    "2": {
+                        "label": "I said negative things about myself."
+                    },
+                    "5": {
+                        "label": "I dominated the situation."
+                    },
+                    "4": {
+                        "label": "I displayed ambition."
+                    },
+                    "7": {
+                        "label": "I engaged in physical activity."
+                    },
+                    "6": {
+                        "label": "I showed high enthusiasm and a high energy level."
+                    },
+                    "9": {
+                        "label": "I was reserved and unexpressive."
+                    },
+                    "8": {
+                        "label": "I concentrated on or worked hard at a task."
+                    }
+                },
+                "options": {
+                    "extremelyUnchar": "Extremely Uncharacteristic",
+                    "quiteUnchar": "Quite Uncharacteristic",
+                    "fairlyUnchar": "Fairly Uncharacteristic",
+                    "somewhatUnchar": "Somewhat Uncharacteristic",
+                    "neutral": "Neither Characteristic nor Uncharacteristic",
+                    "somewhatChar": "Somewhat Characteristic",
+                    "fairlyChar": "Fairly Characterstic",
+                    "quiteChar": "Quite Characteristic",
+                    "extremelyChar": "Extremely Characteristic"
+                },
+                "label": "2. Please rate your behavior at 7 pm/10 am yesterday."
+        },
+            "5": {
+                "items": {
+                    "56": {
```

Comment:
I can double check but I think my python conversion script may not keep the ordering, so having those number keys would be helpful because I could then use `sort_keys` as Andy mentioned.